index.html

```diff
@@ -59,7 +59,7 @@
         <h3>Usability</h3>
 
         <details>
-          <summary>Sandstorm is the easiest way there has ever been to run a server. Install apps as easily as on your phone.</summary>
+          <summary>Install web apps like EtherCalc or WordPress with a click. Use our hosting, or install the platform easily on your own Linux server.</summary>
```

Comment:
Apparently Preeya (one of my reviewers) is much happier with them being four lines long.

Also, I think nearly no one will click on the things below, so I think we should have some fairly high clarity things "above the fold" (in this case, pre-click).